src/userdev/java/net/minecraftforge/gradle/userdev/UserDevPlugin.java

```diff
@@ -329,11 +331,12 @@ public void apply(@Nonnull Project project) {
             Utils.createRunConfigTasks(extension, extractNatives, downloadAssets, createSrgToMcp);
         });
 
-        project.afterEvaluate(projectAfter -> projectAfter.getTasks().withType(JarJar.class).configureEach(jarJar -> {
+        project.getTasks().withType(JarJar.class).configureEach(jarJar -> {
             if (jarJar.isEnabled()) {
+                LOGGER.info("Creating reobfuscation task for JarJar task: " + jarJar.getName());
```

Comment:
Does this log print every time? Or is INFO hidden by default. Also this should use {} pattern.

Reply:
Info is hidden by default.
You need to enable it either with `--info` or by setting the property: `org.gradle.logging.level` to `info` or lower.